extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -91,7 +91,8 @@ var NodeChatSocketWrapper = $.createClass(Observable, {
 
 		};
 
-		this.proxy(callback, this)('name=' + encodedWgUserName + '&key=' + wgChatKey + '&roomId=' + this.roomId + '&serverId=' + this.serverId);
+		this.proxy(callback, this)('name=' + encodedWgUserName + '&key=' + wgChatKey + '&roomId=' + this.roomId
```

Comment:
TODO: add comment above to remember to remove this param: https://wikia-inc.atlassian.net/browse/SUS-432